core/kt-osrd-sim-infra/src/main/kotlin/fr/sncf/osrd/sim_infra/api/InterlockingInfra.kt

```diff
@@ -108,8 +110,8 @@ fun RoutingInfra.getRouteEntry(route: RouteId): DirDetectorId {
     return getZonePathEntry(getRoutePath(route).first())
 }
 
-@JvmName("getRouteExit")
-fun RoutingInfra.getRouteExit(route: RouteId): DirDetectorId {
+@JvmName("getBlockExit")
+fun RoutingInfra.getBlockExit(route: RouteId): DirDetectorId {
```

Comment:
This rename must be a mistake. getRouteExit gets the exit detector or a route, blocks have nothing to do with it

Reply:
Definitely a mistake, I must have accidentally used a fancy IDE feature.

@Erashin did catch the name clash but I fixed it the wrong way around...